src/components/Navbar/Navbar.jsx

```diff
@@ -55,44 +74,92 @@ function NavBar() {
               color={colors.buttonColor}
               backgroundColor="#A96CDE"
               border={`1px solid ${colors.buttonBorderColor}`}
-              _hover={{ 
-                backgroundColor: colors.buttonHoverBgColor 
+              _hover={{
+                backgroundColor: colors.buttonHoverBgColor,
               }}
               size="sm"
               ml={4} // Distance away from Yapper Logo
             >
-              {colorMode === 'light' ? <MoonIcon /> : <SunIcon />}
+              {colorMode === "light" ? <MoonIcon /> : <SunIcon />}
             </Button>
           </Tooltip>
         </Flex>
         {isLargerThanMobile ? (
           <Flex flex="2" justify="flex-end">
             {/* For Seekers */}
```

Comment:
don't know if it's just me, but everytime i refresh the page, the navigation links and logout on navbar for both seeker and employer don't show, might be a jwt or fetching problem